src/xcode/ENA/ENA/Source/Scenes/FileScanner/__tests__/FileScannerCoordinatorTests.swift

```diff
@@ -3,50 +3,20 @@
 //
 
 import XCTest
+import PDFKit
+import PhotosUI
 @testable import ENA
 
 class FileScannerCoordinatorTests: XCTestCase {
 
-	func test_When_finishedPickingImage_Then_dismissViewControllerIsCalled() {
-		let viewControllerSpy = ViewControllerSpy()
-		let viewModel = FileScannerViewModelStub()
-		let fileScannerCoordinator = FileScannerCoordinator(
-			viewControllerSpy,
-			fileScannerViewModel: viewModel,
-			qrCodeFound: { _ in },
-			noQRCodeFound: { }
-		)
-		fileScannerCoordinator.start()
-
-		viewModel.finishedPickingImage?()
-
-		XCTAssertTrue(viewControllerSpy.dimissCalled)
-	}
-
-	func test_When_processingStarted_Then_activityIndicatorIsShown() {
-		let viewControllerSpy = ViewControllerSpy()
-		let viewModel = FileScannerViewModelStub()
-		let fileScannerCoordinator = FileScannerCoordinator(
-			viewControllerSpy,
-			fileScannerViewModel: viewModel,
-			qrCodeFound: { _ in },
-			noQRCodeFound: { }
-		)
-		fileScannerCoordinator.start()
-
-		viewModel.processingStarted?()
-
-		XCTAssertTrue(viewControllerSpy.viewSpy.addSubViewCalled)
```

Comment:
ViewSpy can be removed. Its not used anymore as far as I see.

Reply:
Still needed for other tests